sympy/utilities/lambdify.py

```diff
@@ -336,6 +336,17 @@ def lambdify(args: Iterable, expr, modules=None, printer=None, use_imps=True,
         (if ``args`` is not a string) - for example, to ensure that the
         arguments do not redefine any built-in names.
 
+    cse : bool, None or callable, optional
+        Large expressions can be computed more efficiently when
+        common subexpressions are identified and precomputed before
+        being used multiple time. Finding the subexpressions will make
+        creation of the 'lambdify' function slower, however.
+
+        Whether or not common subexpression elimination should be performed on
+        the expressions. This can potentially improve performance of the generated
+        function. When ``True``, then ``sympy.simplify.cse`` is used, the user
+        may pass their own function matching the signature of said function.
```

Comment:
```suggestion
        When ``True``, the default ``sympy.simplify.cse`` is used, otherwise the user
        may pass a function matching the signature ``cse``.
```